File: C#/Iniciante/1013 - O Maior.cs
/*
* Autor: Pierre Vieira
* Data de submissão:
*/
using System;
using System.Globalization;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            int a, b, c;
            int maior1, maior;
            string[] linha = Console.ReadLine().Split(' ');
            a = int.Parse(linha[0]);
            b = int.Parse(linha[1]);
            c = int.Parse(linha[2]);
            maior1 = (a + b + Math.Abs(a - b)) / 2;
            maior = (maior1 + c + Math.Abs(maior1 - c)) / 2;
            Console.WriteLine($"{maior} eh o maior");
        }

    }
}

File: Java/Iniciante/1013 - O Maior.java
/*
* Autor: Pierre Vieira
* Data de submissão: 04/03/2020 12:31:07
*/
import java.util.Scanner;

import static java.lang.Math.abs;
public class Main {

    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        int a, b, c, maior;
        a = teclado.nextInt();
        b = teclado.nextInt();
        c = teclado.nextInt();
        maior = retorna_maior(a, b);
        maior = retorna_maior(maior, c);
        System.out.println(maior+" eh o maior");
    }

    private static int retorna_maior(int a, int b) {
        return (a+b+abs(a-b))/2;
    }

}
